src/bin.rs

```diff
@@ -0,0 +1,73 @@
+use std::io;
+use std::io::Read;
+
+use anyhow::{Context, Result};
+use clap::{App, Arg};
+use serde_json;
+use serde_json::Value;
+
+use jsonlogic_rs;
+
+fn configure_args<'a, 'b>(app: App<'a, 'b>) -> App<'a, 'b> {
+    app.version(env!("CARGO_PKG_VERSION"))
+        .author("Matthew Planchard <msplanchard@gmail.com>")
+        .about(
+            "Parse JSON data with a JsonLogic rule.\n\
```

Comment:
would it be easier to read if we make about a `r#` multiline string?

Reply:
I don't like how they have to go all the way over to the left side of the screen 🤷‍♂️